mozilla-mobile/android-components/components/feature/prompts/src/main/res/values/strings.xml

```diff
@@ -132,9 +132,13 @@
     <!-- Title of the Identity Credential provider dialog choose. -->
     <string name="mozac_feature_prompts_identity_credentials_choose_provider">Choose a login provider</string>
     <!-- Title of an account picker dialog for identity credentials. -->
-    <string name="mozac_feature_prompts_identity_credentials_choose_account">Choose an account</string>
+    <string name="mozac_feature_prompts_identity_credentials_choose_account" moz:removedIn="119" tools:ignore="UnusedResources">Choose an account</string>
+    <!-- Title of an account picker dialog for identity credentials. The %1$s will be replaced with the name of the provider -->
```

Comment:
What are the possible values for %1%s?

Reply:
@titooan: Could you give some examples here? I assumed these were for emails but may be wrong.

Reply:
I would expect it to be Apple/Google, but I wonder if we're using this also for Mozilla.